e2e-tests/popup/settings/backup secret phrase.spec.ts

```diff
@@ -0,0 +1,111 @@
+import {
+    DEFAULT_FIRST_ACCOUNT,
+    DEFAULT_SECOND_ACCOUNT,
+    vaultPassword
+  } from '../../constants';
+  import { popup, popupExpect } from '../../fixtures';
+ 
+ 
+  //TODO
+  // add check to comparedisplayed phrase with the one used in onboarding
+   popup.describe('Popup UI: back up secret phrase', () => {
+    popup(
+      "should display a secret phrase after providing password and copy phrase",
+      async ({ popupPage, unlockVault, providePassword }) => {
+        await unlockVault();
+         await popupPage.getByTestId('menu-open-icon').click();
+        await popupPage.getByText('Back up your secret recovery phrase').click();
+         await providePassword();
+ 
+ 
+        await popupExpect(
+          popupPage.getByRole('heading', { name: 'Back up your secret recovery phrase' })
+        ).toBeVisible();
+         await popupPage.getByText('Click to reveal secret recovery phrase').click();
+        await popupPage.getByText('Copy secret recovery phrase').click();
+       
+ 
+ 
+        await popupExpect(
+            popupPage.getByText(
+              'Copied to clipboard'
+            )
+          ).toBeVisible();
+      }
```

Comment:
I just wanted to let you know that you didn't check if the secret phrase was displayed and if it was copied.
You test if the text changed after the user clicks on the copy text